dsl_test.go

```diff
@@ -45,9 +110,36 @@ func TestBuilder(t *testing.T) {
 			SetLicense(ogen.NewLicense().
 				SetName(ex.Info.License.Name).
 				SetURL(ex.Info.License.URL),
-			),
+			).
+			SetVersion(ex.Info.Version),
+		).
+		AddServers(
+			&ex.Servers[0],
+			ogen.NewServer().
+				SetDescription(ex.Servers[1].Description).
+				SetURL(ex.Servers[1].URL),
+		).
+		AddPaths(
+			ogen.NewPath(pathWithID).
+				SetDescription(ex.Paths[pathWithID].Description).
+				SetGet(ogen.NewOperation().
+					AddTags(ex.Paths[pathWithID].Get.Tags...).
+					SetDescription(ex.Paths[pathWithID].Get.Description).
+					SetOperationID(ex.Paths[pathWithID].Get.OperationID).
+					AddParameters(
+						ogen.NewParameter().
+							InPath().
+							SetName(ex.Paths[pathWithID].Get.Parameters[0].Name).
+							SetDescription(ex.Paths[pathWithID].Get.Parameters[0].Description).
+							SetRequired(true),
+						ogen.NewParameter().SetRef(authQ.LocalRef()),
+						ogen.NewParameter().SetRef(authH.LocalRef()),
+						ogen.NewParameter().SetRef(csrf.LocalRef()),
```

Comment:
Same here. Consider changing the `Parameter` to an interface (`NamedParam`/`RefParam`/`SimpleParam`), or provide other way that is less verbose to use them here.

```go
AddParameters(
    ogen.NewParamater()...,
    authq.RefParam(),
    // ...
)

// Or using interfaces.
AddParameters(
    ogen.NewParamater()...,
    authq,
    csrf,
    // ...
)
```